File: RemoteSigner/Models/ArgumentModels/GPGDecryptedDataReturn.cs
using System;
namespace RemoteSigner.Models.ArgumentModels {
    public class GPGDecryptedDataReturn {
        public String FingerPrint { get; set; }
        public String Base64Data { get; set; }
        public String Filename { get; set; }
        public bool IsIntegrityProtected { get; set; }
        public bool IsIntegrityOK { get; set; }
    }
}

File: pkg/models/GPGDecryptedDataReturn.go
package models

type GPGDecryptedData struct {
	FingerPrint          string `example:"C1CF31FB8C2A8B59"`
	Base64Data           string `example:"SGVsbG8gd29ybGQK"`
	Filename             string `example:"hello world.txt"`
	IsIntegrityProtected bool   `example:"false"`
	IsIntegrityOK        bool   `example:"false"`
}
